Control/DeepSeq.hs

```diff
@@ -3,6 +3,9 @@
 #if __GLASGOW_HASKELL__ >= 702
```

Comment:
FYI, I'm seriously considering dropping support for GHC < 7.4 with the next `deepseq` release to get rid of all this CPP curft